public/app/core/components/AccessControl/Permissions.tsx

```diff
@@ -266,6 +272,16 @@ const setPermission = (
 ): Promise<void> =>
   getBackendSrv().post(`/api/access-control/${resource}/${resourceId}/${type}/${typeId}`, { permission });
 
+const addTeamLBACWarnings = (teams: number[], items: ResourcePermission[]) => {
+  return items.map((item) => {
+    if (item.permission !== 'Admin' && (!item.teamId || !teams.includes(item.teamId))) {
+      item.warning = `Warning: permission does not have corresponding LBAC rule configured.
+      Users might have full access to the data.`;
```

Comment:
We should [translate](https://github.com/grafana/grafana/blob/main/contribute/internationalization.md) this